default.py

```diff
@@ -32,15 +35,17 @@ def testsites(sites, test_type=None, show_reviews=False, only_test_untested_last
 
         try:
             if test_type == 2:
-                the_test_result = check_four_o_four(website)
+                from tests.page_not_found import run_test
```

Comment:
Clever! ...I wonder if it'd be possible to build it so that multiple tests could be run, say 6 AND 7? But that's more a feature idea than a review comment really...